Auto-discover message class handle by handler (#4)
Wynd-Lab/nihilus-symfony-resolver: doc/debug.md

### Debug

> An order is available to understand how your query / command will be handled. which middlewares will be used and which handler will support the message

Reviewer: ```suggestion
A Symfony command is available to understand how your query / command will be handled. which middlewares will be used and which handler will support the message
```
